src/AutoRest.CSharp.V3/Output/Models/Requests/MultipartRequestBody.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+
+namespace AutoRest.CSharp.V3.Output.Models.Requests
+{
+    internal class MultipartRequestBody : RequestBody
+    {
+        public ReferenceOrConstant[] Value { get; }
```

Comment:
I think we need an explicit MultipartRequestBodyPart class that would have a Name property.

When we add the method parameter customization the actual parameter name might be different than multipart part name